<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_image</class>
 <widget class="QWidget" name="tab_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Initial X:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Initial Y:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Img Area Y:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Img Area X:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Loop Mode:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Blocking:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Delay per pixel moved:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select Entry:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="init_x_spinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Dest X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="destx_spinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="init_y_spinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Dest Y:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="desty_spinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>-999</number>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="img_area_x">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>320</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Img Area W:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="img_area_w">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>320</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="img_area_y">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>240</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Img Area H:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="img_area_h">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>240</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QSpinBox" name="delay_spinBox">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="1">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>No Loop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Continuous Loop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Auto-Reverse Loop</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QCheckBox" name="blocking_checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="filename_comboBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="name_textEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="select_comboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="add_Button">
         <property name="text">
          <string>Add New Entry</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Reset Size</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/toolbar_icons/task-recurring.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>148</width>
          <height>383</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="ImageAreaPreview" name="image_preview_widget" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>320</width>
           <height>334</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageAreaPreview</class>
   <extends>QWidget</extends>
   <header>scenes/imageareapreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
